### ISSUE TITLE
Learning rate schedule 

### ISSUE THREAD

**Issue by junhyukk** [InterDigitalInc/CompressAI#98]
> Unless specified otherwise, networks were trained for 4-5M steps on 256x256 image patches randomly extracted and cropped from the Vimeo90K dataset [xue2019video]."

> Models were trained with a batch size of 16 or 32, and an initial learning rate of 1e-4 for approximately 1-2M steps. The learning rate of the main optimizer is then divided by 2 when the evaluation loss reaches a plateau (we use a patience of 20 epochs). This can be implemented by using PyTorch ReduceLROnPlateau learning rate scheduler.

I have several questions about the details of model training. 

1. Have you tried smaller batch size of 16 (e.g., 8)? I'm wondering that using the smaller one has a bad effect on final performance. 

2. The initial learning rate of 1e-4 was constant until the "predefined" steps (e.g. 2M), and after the steps, you started to use the evaluation loss in order to reduce the learning rate? 
or you started to use the learning scheduling from scratch, and "empirically" observed most models started to reduce their learning rate around 1~2M steps?

3. When is the stop point? 
If you used "Predefined" (e.g., 5M) steps, the learning rate scheduling (i.e., divided by 2 based on the evaluation loss) continued until the predefined step? or Did the learning rate decrease only once? 

4. What's the learning rate schedule for aux_optimizer? 
Did you use the constant learning rate (i.e., 1e-3) during the whole training process?  

Thank you in advance. :)

**Comment by fracape** [comment]
Hello. This repo aims at providing the implementation in pytorch of the required layers and modules to build and compare E2E image compression models and traditional methods. We just provide pre-trained models for reference, we found that the results closely match  those provided in the literature, but don't claim we found the best hyper parameters for each of them. Since this is not directly a bug or issue, I would recommend to use the "discussions" section to share training settings so the community (and potentially original authors) can comment. Thanks!
